resources/js/Shared/Layout/Nav.vue

```diff
@@ -128,97 +129,130 @@ defineExpose({
+              <label class="mb-1 block text-sm font-semibold text-gray-800">{{ __('Name') }}</label>
+              <input v-model="registerForm.name" type="text" class="w-full rounded-lg border-blumilk-200 py-3 md:p-2"
+                     required
+              >
               <ErrorMessage :message="registerForm.errors.name" />
             </div>
 
             <div>
-              <label class="mb-1 block text-sm font-semibold text-gray-800">Email</label>
-              <input v-model="registerForm.email" type="email" class="w-full rounded-lg border-blumilk-200 py-3 md:p-2" required>
+              <label class="mb-1 block text-sm font-semibold text-gray-800">{{ __('Email') }}</label>
+              <input v-model="registerForm.email" type="email" class="w-full rounded-lg border-blumilk-200 py-3 md:p-2"
+                     required
+              >
               <ErrorMessage :message="registerForm.errors.email" />
             </div>
             <div>
-              <label class="mb-1 block text-sm font-semibold text-gray-800">Password</label>
-              <input v-model="registerForm.password" type="password" class="w-full rounded-lg border-blumilk-200 py-3 md:p-2" required>
+              <label class="mb-1 block text-sm font-semibold text-gray-800">{{ __('Password') }}</label>
+              <input v-model="registerForm.password" type="password"
+                     class="w-full rounded-lg border-blumilk-200 py-3 md:p-2" required
+              >
             </div>
             <div>
-              <label class="mb-1 block text-sm font-semibold text-gray-800">Confirm password</label>
-              <input v-model="registerForm.password_confirmation" type="password" class="w-full rounded-lg border-blumilk-200 py-3 md:p-2" required>
+              <label class="mb-1 block text-sm font-semibold text-gray-800">{{ __('Confirm password') }}</label>
+              <input v-model="registerForm.password_confirmation" type="password"
+                     class="w-full rounded-lg border-blumilk-200 py-3 md:p-2" required
+              >
               <ErrorMessage :message="registerForm.errors.password" />
             </div>
             <div class="flex w-full md:w-fit">
-              <button type="submit" class="w-full rounded-lg bg-blumilk-500 p-4 font-semibold text-white hover:bg-blumilk-600 md:py-2">
-                Sign up
+              <button type="submit"
+                      class="w-full rounded-lg bg-blumilk-500 p-4 font-semibold text-white hover:bg-blumilk-600 md:py-2"
+              >
+                {{ __('Sign up') }}
               </button>
             </div>
           </form>
           <button :disabled="registerForm.processing" class="mt-6 text-xs font-light" @click="toggleAuthOption">
-            Already have an account? <span class="font-normal">Log in</span>
+            {{ __('Already have account') }} <span class="font-normal">{{ __('Log in') }}</span>
```

Comment:
I think it should stay as it was before. Remember about translation.

Reply:
Fixed